### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by almet**
This is based atop the hawk branch.

Update the documentation with the new APIs and hawk support.

### CODE REVIEW

File: docs/source/usage.rst [hunk 1]
```diff
@@ -105,32 +164,78 @@ Pushing data
 **POST /models/{modelname}/records**
 **PUT /models/{modelname}/records/{id}**
 
-Now that we defined the schema, we want to push some real data there::
+Now that we defined the schema, we want to push some real data there!::
```
Comment (Natim) [inline docs/source/usage.rst:167]: How does `!::` works? Will it display `!:` ?

**Comment by Natim** [comment]
Good start :)

File: docs/source/usage.rst [hunk 2]
```diff
@@ -51,53 +82,81 @@ like this:
                 "label": "is it done or not"
             }
          ]
-      }
-    }'
+      }}' > definition
 
-    curl -XPUT http://localhost:8000/models/todo -d "${model}" -u admin@example.com:apikey
+    http put http://localhost:8000/models/todo @definition --verbose --auth-type=hawk --auth='ad37fc395b7ba83eb496849f6db022fbb316fa11081491b5f00dfae5b0b1cd22:'
```
Comment (leplatrem) [inline docs/source/usage.rst:88]: put uppercase ?

File: docs/source/usage.rst [hunk 3]
```diff
@@ -1,38 +1,69 @@
...
-Let's say we want to have a daybed managed todo list. First, we put
-a definition under the name "todo".
+Let's say we want to have a daybed-managed todo list. Follow the steps and you
+will have a grasp of how daybed works.
+
+To ease testing, you can use `httpie <https://github.com/jkbr/httpie>`_ in
+order to make requests. Examples of use with httpie are provided when possible.
 
 
+In order to authenticate with `Hawk <https://github.com/hueniverse/hawk>`_,
```
Comment (leplatrem) [inline docs/source/usage.rst:14]: httpie and Hawk ?

**Comment by Natim** [comment]
I have merged a bit too early sorry about that.

**Pull request merged**